pkg/util/kubernetes/apiserver/controllers.go

```diff
@@ -154,11 +150,6 @@ func startAutoscalersController(ctx ControllerContext, c chan error) {
 	autoscalersController.RunControllerLoop(ctx.StopCh)
 }
 
-// registerServicesInformer registers the services informer.
-func registerServicesInformer(ctx ControllerContext, c chan error) {
```

Comment:
this is only used for the controller above. We should be able to delete this cc @xlucas